Video overview

colorfield opened the pull request:
Site building, backend/frontend dev for #6 
As there is already a lot going on with this PR, created proposed follow ups in the original issue.

colorfield committed:
Image and video site building

- Restricted html description for image and video
- Set permissions for authenticated (limit to restricted html/nofollow/...)
- license and rights fields for image
- Tweak default, full and teaser view modes

colorfield committed:
Add video category filter

colorfield committed:
Set back node title on teaser, preprocess category field, card styling (wip)

Using here the node title instead of the media name as
- media name is automatically set for video only (not for image)
- the user fills the node title while contributing so expects this one to be displayed.

colorfield committed:
Margin fix

colorfield committed:
Require search api, facets, views infinite scroll

colorfield committed:
Search api setup

colorfield committed:
Change theme views reference

colorfield committed:
Add videos and images category facets

colorfield committed:
Render the title on the node template

colorfield committed:
Append the author card to image and video nodes

Initially it was the media author, but there is a 1 to 1 reference between media and node, a possible issue could be that the media changes afterwards?

colorfield commented:
@dan2k3k4 while moving to search api, these conditions have been adapted https://github.com/drupal-celebrations/celebrate-drupal-9/pull/29/commits/395900524d1fd0c238da83da072b03936f7029ed#diff-994789ec470bc8c7745ba5ef2d8a3061R40-R42 but i might miss other classes for the columns, it would be great if you could have a look. Thanks!

colorfield committed:
Set back debug class

colorfield committed:
User card

colorfield committed:
Set category background and text color from the Term color field

colorfield committed:
Add icons for author links

colorfield committed:
Fix side effect of views infinite scroll on columns display

dan2k3k4 commented on web/modules/custom/user_registration/images/drupal-evergreen-logo.svg:
```diff
@@ -0,0 +1,12 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 42.2 55.5">
```
As we noticed, we'd need to tweak this to full numbers like: `viewBox="0 0 44 58"`

colorfield replied:
Thanks! Oddly here it does not seem to have any influence, perhaps it was related to the path.
![Screenshot 2020-05-24 at 01 22 36](https://user-images.githubusercontent.com/525003/82742349-2bc3e800-9d5d-11ea-8f8f-5face1cf7a93.png)

colorfield replied:
Still rounded the viewBox https://github.com/drupal-celebrations/celebrate-drupal-9/pull/29/commits/ec5017984e6c334ff9c3450c8894f9d770267088

colorfield committed:
Make use of the poster on video teaser to link to the node

- Queuing thumbnails is disabled, we might need to check perfs but otherwise the user won't see the thumbnail until the next cron run
- Responsive image style needs work

colorfield committed:
Round svg viewBox

Commit:
Add patch

dan2k3k4 reviewed (approved):
Some nitpicks like remove `field_cc_license` but to save time I'll get this merged and we can remove things we don't need after
